install_ffmpeg.sh

```diff
@@ -47,6 +48,7 @@ else
     echo "clang detected, building with GPU support"
 
     EXTRA_FFMPEG_FLAGS="--enable-cuda --enable-cuda-llvm --enable-cuvid --enable-nvenc --enable-decoder=h264_cuvid --enable-filter=scale_cuda,signature_cuda --enable-encoder=h264_nvenc"
+    # LIBTENSORFLOW_SUPPORT="gpu"
```

Comment:
Can we confirm that CPU version of Tensorflow is required when there's no GPU? Official site states something like 'with GPU support' for GPU version, which doesn't mean it won't work in CPU mode, and same is my personal experience. Would be great to check whether we can just use GPU version in both cases.

Reply:
I have tested with GPU version of Tensorflow on the device that does not have GPU. Actually, GPU version is working for both cases.